### PR TITLE
Printpage updates

### PR DESCRIPTION
**Pull request by zeddreactor**
Addressing issues #699 #697 #698

### CODE REVIEW

File: src/web/components/PatientView/PrintRecord/PrintPathologyForm.js [hunk 1]
```diff
@@ -50,39 +53,62 @@ const Block = ({
...
+                      </span>
+                    </span>
+                  </td>
+                </tr>
+              );
+            }
+          }
+          return elements;
+        }
+         //  (
```
Comment (tuttieee) [inline src/web/components/PatientView/PrintRecord/PrintPathologyForm.js]: Can you please remove these lines?

File: src/web/components/PatientView/PrintRecord/PrintPathologyForm.js [hunk 2]
```diff
@@ -117,13 +147,31 @@ export default ({
...
-        </div>
+      {
+        (style.slice(0, 2)).map((element, index) => (
+          <div className="column" key={index}>
+            <Block data={record} block={element} />
+          </div>
+        ))}
       </div>
+      {(() => {
+        let testtype;
```
Comment (tuttieee) [inline src/web/components/PatientView/PrintRecord/PrintPathologyForm.js]: (There is no problem. This is just an advice or suggestion)

It seems a variable `testtype` can be moved to inside the for loop like this:

```
...
for (let i = 2; i < style.length; i += step) {
  const testtype = (...);
}
```

This makes the scope of the variable narrow.

File: src/web/components/PatientView/PrintRecord/PrintPathologyForm.js [hunk 3]
```diff
@@ -117,13 +147,31 @@ export default ({
...
+        const testsColumn = [];
+        const step = 3;
+        for (let i = 2; i < style.length; i += step) {
+          testtype = (<div className="columns" key={i}>
+          {((style.slice(i, i + step)).map((element, index) => (
```
Comment (tuttieee) [inline src/web/components/PatientView/PrintRecord/PrintPathologyForm.js]: Outer parentheses seems unnecessary

```js
style.slice(i, i + step).map((element, index) => ())
```

File: src/web/components/PatientView/PrintRecord/PrintPathologyForm.js (hunk 2 above)
Comment (zeddreactor) [reply src/web/components/PatientView/PrintRecord/PrintPathologyForm.js]: Indeed! Thanks, I missed that :) was planning to make it used after the loop initially and forgot to narrow the scope.

File: src/web/components/PatientView/PrintRecord/PrintPathologyForm.js (hunk 3 above)
Comment (zeddreactor) [reply src/web/components/PatientView/PrintRecord/PrintPathologyForm.js]: Oops... you are right

**Pull request merged**